src/frontend/src/lib/components/common/QRButton.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+	import { IconQRCodeScanner } from '@dfinity/gix-components';
+</script>
+
+<button type="button" data-tid="qr-code-scanner-button" class="icon-only" on:click|preventDefault>
```

Comment:
- the accessibility labels are missing
- do we need the data-tid?
- do we need the icon-conly class?
- do we need preventDefault given that the button is of type button?

Reply:
- accessibility: will add it
- data-tid: I would leave it, for future E2E testing scopes
- icon-only: not necessary, but it is always good to leave the class definition no? I can remove them if you want
- preventDefault: I would leave it there, because it may happen the same behavior of the Max button, where a keyboard event could trigger it involuntarily

Reply:
- accessibility thx
- data-tid ok
- icon-only is not a class of oisy so let's remove it
- preventDefault ok